Extensions/BitmapText/bitmaptextruntimeobject.ts

```diff
@@ -0,0 +1,296 @@
+namespace gdjs {
+  /** Base parameters for {@link gdjs.BitmapTextRuntimeObject} */
+  export type BitmapTextObjectDataType = {
+    /** The base parameters of the Bitmap Text */
+    content: {
+      /** The opacity of the Bitmap Text */
+      opacity: number;
```

Comment:
`float` instead of `number` (use `float` or `integer` in TypeScript - they are the same thing, but will be useful in the future.)